dolphinscheduler-common/src/test/java/org/apache/dolphinscheduler/common/utils/NetUtilsTest.java

```diff
@@ -99,6 +99,21 @@ public void testIsValidAddress() {
         address = mock(InetAddress.class);
         when(address.getHostAddress()).thenReturn("1.2.3.4");
         assertTrue(NetUtils.isValidV4Address(address));
+        address = mock(InetAddress.class);
+        when(address.getHostAddress()).thenReturn("1.2.3.4:80");
+        assertFalse(NetUtils.isValidV4Address(address));
+        address = mock(InetAddress.class);
+        when(address.getHostAddress()).thenReturn("256.0.0.1");
+        assertFalse(NetUtils.isValidV4Address(address));
+        address = mock(InetAddress.class);
+        when(address.getHostAddress()).thenReturn("127.1");
```

Comment:
'127.1' is a validated IpV4 
https://superuser.com/questions/614001/why-can-i-ping-127-1

Reply:
> '127.1' is a validated IpV4 https://superuser.com/questions/614001/why-can-i-ping-127-1

Hi @ruanwenjun , could we use `InetAddress.isReachable()` to define this IP? because Ipv4 has many format

Reply:
Use `IPAddressUtil.isIPv4LiteralAddress` is enough.

Reply:
> Use `IPAddressUtil.isIPv4LiteralAddress` is enough.

OK. done